cas_implementations/src/eav/file.rs

```diff
@@ -140,7 +141,13 @@ impl EavFileStorage {
                     pathbuf
                         .iter()
                         .last()
-                        .map(|v| eav_filter.check(v.to_string_lossy().to_string().into()))
+                        .and_then(|v| {
+                            v.to_string_lossy()
+                                .to_string()
+                                .try_into()
+                                .ok()
```

Comment:
I'm a bit skeptical of using ok() because we end up losing any possible error information but this is just a string conversion so I am fine with it

Reply:
Why do we need this *cascade of conversions* ( 😉) anyway? Isn't there a better way of doing this?

Reply:
@AshantiMutinta the error case here (coming from `try_into` would be that the string doesn't correspond to a valid attribute. You're right though this would be a hard thing to catch. Maybe it should log a warning message?

@sphinxc0re it's because we have to go from (off the top of my head) `PathBuf` -> `Cow<&str>` -> `String` -> `Result<Attribute, _>`, if you see a shorter path do suggest it!